### ISSUE TITLE
Waveform keyboard seeking does not work

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-frontend#628]
## Description

Seeking on the audio waveform via the keyboard does not work. Distinct from #623 which is only about the `tab` navigation, not the seeking.

## Reproduction

1. Run `main` and visit http://localhost:8443/audio/632d0076-fdbd-4919-8b8f-b498ef246ee8
2. Focus the waveform using the keyboard and attempt to seek the file.
3. Note that it does not work.

## Expectation

The keyboard controls built into the waveform component should work.

## Resolution

- [x] 🙋 I would be interested in resolving this bug.
